src/sezzle-checkout-installment-widget/sezzle-checkout-installment-widget.js

```diff
@@ -647,7 +665,41 @@ function renderInstallmentWidget(checkoutTotal, serviceRegion, currencySymbol){
 		if(sezzleModalClose.length){
 			for(var i = 0; i < sezzleModalClose.length; i++){
 				sezzleModalClose[i].addEventListener('click', closeSezzleModal);
+				sezzleModalClose[i].addEventListener('click', () => {
```

Comment:
Arrow functions have issues in older browsers - recommended to update this line to use `function(){...` format